### PR TITLE
feature/28: Implemented Browser special keys (Escape, Backspace, Enter and Supr)

### PR DESCRIPTION
**Pull request by EduAlberdi**
# PR Details

Added new keys such as Escape, Backspace, Enter and Supr available to call thought Browser class

## Related Issue

#28 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation 
- [ ] I have updated the documentation accordingly (README.md for each widget)
- [ ] I have added tests to cover my changes (at least 1 spec for each widget with the same coverage as the master branch)
- [x] All tests (new and existing) passed
- [ ] A new branch needs to be created from master to evolve previous versions
- [x] Increase version in package.json following [Semantic Versioning](https://semver.org/)
- [x] Add the issue into the right [project](https://github.com/systelab/systelab-components-wdio-test/projects) with the proper status (In progress)

### CODE REVIEW

File: src/wdio/browser.ts [hunk 1]
```diff
@@ -12,9 +12,21 @@ export class Browser {
...
     public static async pressTab(): Promise<void> {
         return browser.keys(['Tab']);
     }
+    public static async pressBackspace(): Promise<void> {
+        return browser.keys(['Backspace']);
+    }
+    public static async pressEnter(): Promise<void> {
+        return browser.keys(['Enter']);
+    }
+    public static async pressSupr(): Promise<void> {
```
Comment (joaquimvila) [inline src/wdio/browser.ts:27]: I would name this one as "pressDelete", considering that "Supr" is the wording of Spanish keyboards, so we should go to a more generic approach

File: src/wdio/browser.ts [hunk 2]
```diff
@@ -12,9 +12,21 @@ export class Browser {
 
 
     // Keyboard special keys
+    public static async pressEsc(): Promise<void> {
+        return browser.keys(['Escape']);
+    }
```
Comment (joaquimvila) [inline src/wdio/browser.ts:17]: A picky one I know, but just to keep the code styling on this file, I miss a blank line between the added methods.

**Review by joaquimvila — APPROVED** [review]
Good job Eduard!
Thanks for your contribution. This new methods would be helpful not only for your assigned project, but also for other CSW projects.

**Comment by Franek75** [comment]
A nice one! (I was on holidays when I was asked for review, so I am just taking a "retrospective look" now)

**Pull request merged**